security/global/base-identities/roles.tf

```diff
@@ -91,7 +65,7 @@ module "iam_assumable_role_oaar" {
   source = "github.com/binbashar/terraform-aws-iam.git//modules/iam-assumable-role?ref=v4.7.0"
 
   trusted_role_arns = [
-    "arn:aws:iam::${var.accounts.root.id}:root"
+    "arn:aws:iam::${var.accounts.management.id}:root"
```

Comment:
@eze-godoy We've roll-back this change (https://github.com/binbashar/le-tf-infra-aws/pull/417) because it mess the leverage SSO setup for the management acocunt.

CC: @marianod92 @angelofenoglio

Reply:
```suggestion
    "arn:aws:iam::${var.accounts.root.id}:root"
```